explorer/check_for_peers.sh

```diff
@@ -0,0 +1,8 @@
+#! bash
+
+PEERS_EXTANT=$(docker ps -a --format '{{.Names}}' -f "name=^peer[0-9]+\.org[0-9]+")
+if [ -z "$PEERS_EXTANT" ]; then
+    echo "Explorer cannot run without network to connect to."
```

Comment:
Looks good, might just want to suggest the right target to run to be totally clear.